static/app/views/issueDetails/groupReplays/useReplaysFromIssue.tsx

```diff
@@ -54,12 +53,12 @@ function useReplayFromIssue({
       name: '',
       version: 2,
       fields: REPLAY_LIST_FIELDS,
-      projects: [Number(group.project.id)],
```

Comment:
Will whatever endpoint that uses this eventview still work for orgs on the team plan?

Reply:
![image](https://user-images.githubusercontent.com/7349258/235160731-27efbcc2-8910-4071-b2ab-1e5407df16a9.png)

looks like no 😢